definition_schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://json-schema.org/draft/2020-12/schema",
+  "$vocabulary": {
+    "https://json-schema.org/draft/2020-12/vocab/core": true,
+    "https://json-schema.org/draft/2020-12/vocab/applicator": true,
+    "https://json-schema.org/draft/2020-12/vocab/unevaluated": true,
+    "https://json-schema.org/draft/2020-12/vocab/validation": true,
+    "https://json-schema.org/draft/2020-12/vocab/meta-data": true,
+    "https://json-schema.org/draft/2020-12/vocab/format-annotation": true,
+    "https://json-schema.org/draft/2020-12/vocab/content": true
+  },
+  "$dynamicAnchor": "meta",
+  "title": "Core and Validation specifications meta-schema",
+  "allOf": [
+    {
+      "$ref": "meta/core"
+    },
+    {
+      "$ref": "meta/applicator"
+    },
+    {
+      "$ref": "meta/unevaluated"
+    },
+    {
+      "$ref": "meta/validation"
+    },
+    {
+      "$ref": "meta/meta-data"
+    },
+    {
+      "$ref": "meta/format-annotation"
+    },
+    {
+      "$ref": "meta/content"
+    }
+  ],
+  "unevaluatedProperties": false,
+  "type": [
+    "object",
+    "boolean"
+  ],
+  "$comment": "This meta schema for the Eiffel Definitions files is a copy of a meta schema with some additions.",
+  "properties": {
+    "_description": {
+      "type": "string"
+    },
+    "_abbrev": {
+      "type": "string"
+    },
+    "_examples": {
+      "type": "array"
+    },
+    "_history": {
+      "type": "array"
+    },
+    "_links": {
+      "type": "object"
+    },
+    "_name": {
+      "type": "string"
+    },
+    "_version": {
+      "type": "string"
+    },
```

Comment:
We should be able to be a bit more specific in these definitions, no? Perhaps you're just testing the waters and keeping the PR as a draft?

Reply:
Maybe a bit more details. E.g. _history is just an array. Could add what type it us under.

Reply:
Fixed in [46b96de](https://github.com/eiffel-community/eiffel/pull/402/commits/46b96defda2c6d7c8e1436cfe800e67d1f0dcaff)